chain/network/src/network_protocol/borsh.rs

```diff
@@ -160,4 +160,4 @@ pub(super) enum PeerMessage {
     VersionedStateResponse(StateResponseInfo),
 }
 #[cfg(target_arch = "x86_64")] // Non-x86_64 doesn't match this requirement yet but it's not bad as it's not production-ready
-const _: () = assert!(std::mem::size_of::<PeerMessage>() <= 1144, "PeerMessage > 1144 bytes");
+const _: () = assert!(std::mem::size_of::<PeerMessage>() <= 1500, "PeerMessage > 1500 bytes");
```

Comment:
not sure what the value of having that and it breaks builds when new fields are added to block or chunk headers, so I'm increasing this value to avoid updating that after each new field is added